docs/adr/0013-self-managed-rancher-cluster.md

```diff
@@ -0,0 +1,58 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [13. Self managed Rancher cluster](#title)
+  - [Context](#context)
+  - [Decision](#decision)
+  - [Consequences](#consequences)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Self managed Rancher cluster
+
+- Status: proposed
+- Date: 2025-01-20
+- Authors: @anmazzotti
+- Deciders:
+
+## Context
+
+There are currently no best practices, documentation, or support to deploy a self managed Rancher cluster.  
+In this scenario, a user such as a platform engineer starts from zero.  
+
+- How can they bootstrap their first management Cluster?
+- How to install Rancher in it?
+- How to install Rancher Turtles and make the cluster also manage itself using CAPI?
+- How can a self managed cluster be provisioned from an air-gapped environment?
+
+## Decision
+
+The proposed solution is to support a [Bootstrap & Pivot](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html#bootstrap--pivot) process, so that the user can first create a temporary management cluster, then use it to bootstrap the Rancher management cluster, and finally pivot all needed resources to it.  
+
+In order to ensure compatibility and sane validation checks, initing and moving resources should be done using the `clusteroperator` CLI.  
```

Comment:
let's reference it for someone new to follow the ADR: https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/cmd/plugin/cmd